src/kv.c

```diff
@@ -27,6 +27,9 @@ static const char *kvSubjectsTmpl           = "$KV.%s.>";
 static const char *kvSubjectsPreTmpl        = "$KV.%s.";
 static const char *kvSubjectsPreDomainTmpl  = "%s.$KV.%s.";
 
+static natsStatus
```

Comment:
Would not need that if you have the static before the public functions. See below.